src/deprecates.jl

```diff
@@ -48,3 +48,15 @@ findat(a::AbstractArray, b::AbstractArray) = findat!(Array{Int}(size(b)), a, b)
 @deprecate df_residual(obj::StatisticalModel) dof_residual(obj)
 
 @deprecate_binding WeightVec Weights
+
+using Base.Random: RangeGenerator
+@deprecate_binding RandIntSampler RangeGenerator
+@deprecate RangeGenerator(K::Int) RangeGenerator(1:K)
```

Comment:
I guess another strategy would be to make a temporary type with a silly name like `UseRangeGeneratorInstead`, I think some of the linspace / float-range deprecations in base were done like that